nav2_system_tests/src/recoveries/backup/CMakeLists.txt

```diff
@@ -0,0 +1,23 @@
+set(test_backup_recovery_exec test_backup_recovery_node)
+
+ament_add_gtest_executable(${test_backup_recovery_exec}
+  test_backup_recovery_node.cpp
+  backup_recovery_tester.cpp
+)
+
+ament_target_dependencies(${test_backup_recovery_exec}
+  ${dependencies}
+)
+
+ament_add_test(test_backup_recovery
+  GENERATE_RESULT_FOR_RETURN_CODE_ZERO
+  COMMAND "${CMAKE_CURRENT_SOURCE_DIR}/test_backup_recovery_launch.py"
+  WORKING_DIRECTORY "${CMAKE_CURRENT_BINARY_DIR}"
+  TIMEOUT 180
```

Comment:
Any reason why the timeout is 180 seconds? Can be reduced right?

Reply:
We had a big discussion about this in another  ticket and we bumped these numbers up for all the tests because of inconsistent results on people's lower powered machines running it locally. I thought it was better to pick something that matches the lowest common denominator (within reason).